src/Resources/config/phpcr-odm.xml

```diff
@@ -5,7 +5,7 @@
 
     <services>
         <!-- Adapters -->
-        <service id="cmf_routing_auto.adapter.phpcr_odm" class="Symfony\Cmf\Bundle\RoutingAutoBundle\Adapter\PhpcrOdmAdapter">
+        <service id="cmf_routing_auto.adapter.phpcr_odm" class="Symfony\Cmf\Bundle\RoutingAutoBundle\Adapter\PhpcrOdmAdapter" public="true">
```

Comment:
why does this service now need to be public? (I guess it's not used by the ORM bundle?)

Reply:
You are right. I've forgotten to remove this.